CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## [v0.28.0](https://github.com/mongodb/go-client-mongodb-atlas/tree/v0.28.0) (May 29, 2023)
```

Comment:
A question that is not related to this PR but only for my own knowledge: why did we add this feature to the manual SDK instead of using the auto-generated one?